src/App.tsx

```diff
@@ -1,4 +1,8 @@
+import { IntlProvider } from "react-intl";
 import { Sidebar, type DataSource } from "./components";
+import { messages } from "./i18n";
+
+const userLocale = "en";
```

Comment:
I'm thinking if we can also link it with the `<html>` tag in the `index.html` file.

Reply:
I am not sure what you mean here, can you please elaborate?